src/components/chartRepo/ChartRepo.tsx

```diff
@@ -249,9 +270,47 @@ function ChartForm({ id = null, name = "", active = false, url = "", authMode =
         }
     }
 
+    function toggleIsPublicChartType(e) {
+        customState.username = { value: '', error: '' }
+        customState.password = { value: '', error: '' }
```

Comment:
Don't mutate the state. Use state setter to set new values

Reply:
done